src/ui/include/mainwindow.h

```diff
@@ -256,16 +254,13 @@ private slots:
     int                 saveAs(EditorTabWidget *tabWidget, int tab, bool copy);
     QUrl                getSaveDialogDefaultFileName(EditorTabWidget *tabWidget, int tab);
     void                setupLanguagesMenu();
-    void                transformSelectedText(std::function<QString (const QString &)> func);
+    void                transformSelectedText(QString (*func)(const QString&));
```

Comment:
It didn't make sense to have the overhead for something that doesn't need any captures to begin with.